packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -287,6 +243,7 @@ reactivate-success-copy = Thanks! You're all set.
 reactivate-success-button = Close
 
 ## subscription item
+## $period (Date) - The last day of product access
```

Comment:
Can you document `$name` here as well? Or next to `sub-item-cancel-msg` and `sub-item-cancel-confirm`.